src/jack.py

```diff
@@ -1979,8 +1980,7 @@ class OwnPort(Port):
     """
 
     def __init__(self, port_ptr, client):
-        Port.__init__(self, port_ptr)
-        self._client = client
+        Port.__init__(self, port_ptr, client)
```

Comment:
I think you can just remove the whole `__init__()` method from `OwnPort`, right?